pkg/actuators/machine/machine.go

```diff
@@ -302,7 +302,22 @@ func (ms *machineScope) delete() error {
 		}
 		return errors.Wrap(err, "error finding VM by name")
 	}
-	return ms.ovirtClient.RemoveVM(vm.ID(), ovirtC.ContextStrategy(ms.Context))
+	if vm == nil {
```

Comment:
This case is already handled in line 300 above. Why is this check here again?

Reply:
sure , with this PR https://github.com/oVirt/go-ovirt-client/pull/181 we wont need it, removing